### DISCUSSION TITLE
New release (2.6)

### DISCUSSION THREAD

**Discussion by sylvaincombes** [azuyalabs/yasumi#299] [General]
Hello and many thanks for your hard work, can we expect some tagged release, as the latest is 2.5.0 and was released on 30 January 2022 ?

Thank you

**Comment by stelgenhof** [comment]
Hi @sylvaincombes Definitely! I am trying to collect a good number of changes and fixes for a next release. Some provided changes are taking a bit longer than expected. But for sure will make a release soon.
Thanks for the reminder!

**Comment by stelgenhof** [comment]
Currently I am fixing some issues and merging outstanding PR's.

**Comment by stelgenhof** [comment]
Finished most of the changelog updates. Still some more editing to do...

**Comment by stelgenhof** [comment]
Released a new version: 2.6 (See #313)  is here!
